applications/StructuralMechanicsApplication/custom_conditions/base_load_condition.h

```diff
@@ -207,6 +207,20 @@ class KRATOS_API(STRUCTURAL_MECHANICS_APPLICATION)  BaseLoadCondition
         ProcessInfo& rCurrentProcessInfo 
         ) override;
 
+     /**
+     * this function is designed to make the element to assemble an rRHS vector
+     * identified by a variable rRHSVariable by assembling it to the nodes on the variable
+     * rDestinationVariable.
+     * @param rRHSVector: input variable containing the RHS vector to be assembled
```

Comment:
Remove the `:` Doxygen just recognizes the @param in case of no `:`, this can be generalized to all @KratosMultiphysics